DEVELOPING.md

```diff
@@ -0,0 +1,51 @@
+Guidelines for Developers of OpenSim-Core
+===========================================
+OpenSim is a community resource that is housed in the OpenSim-Core repository.
+
+This specific document addresses issues that come up during development and maintenance of OpenSim code base, that may not be relevant to end users because it's not in release code yet or of histroical relevance only but would be 
+valuable to new developers or advanced users. For general contribution guidelines please consult [CONTRIBUTING.md](https://github.com/opensim-org/opensim-core/blob/master/CONTRIBUTING.md)
+
+Contents:
+
+- [Backward Compatibility](#backward-compatibility)
+
+
+Backward Compatibility
```

Comment:
P.S. That's done in Markdown by putting a line of dashes `--------------` underneath the heading.